Documentation/network/servicemesh/external-ingress-policy.rst

```diff
@@ -0,0 +1,71 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_external_ingress_policy:
+
+*************************
+External Lock-down Policy
+*************************
+
+The previous example doesn't include any network policy, so all the external traffic is allowed by default.
```

Comment:
You're referring to a previous example here, but this is the first example shown in the document. This might be confusing for users who find this page via Google or for users who don't read the previous example. Can you make this section more independent, or add it to the document that has the previous example you are referring to?